datasetinsights/estimators/faster_rcnn.py

```diff
@@ -79,7 +79,7 @@ def __init__(
         self._init_distributed_mode()
         self.no_cuda = no_cuda
         self._init_device()
-        self.writer = SummaryWriter(writer.logdir, write_to_disk=is_master())
+        self.writer = SummaryWriter(logdir)
```

Comment:
Does this `SummaryWriter` solve the `connection error` issue?

Reply:
Yes, it does.

Reply:
@Saurav-D This seems to be enable tensorboard logs for non-master training processes. What's the mechanism behind the scene that prevents non-master training processes from writing to tensorboard?

Reply:
If the answer is "yes, pytorch tensorboard will handle this behind the scene", this should be moved to `create_estimator` in base.py

Reply:
Sure, I can create a PR for this.